lisa_commands/commands.py

```diff
@@ -252,14 +252,53 @@ def image_me(query, _slack_event):
         return "From Google: " + request.google_image_search(query)
     return "From Bing: " + request.bing_image_search(query)
 
-def kill_me(_query, _slack_event):
+def kill_me(_query, slack_event):
     """
     query - query str (unused for this function)
-    slack_event - A dict of slack event information (unused for this function)
+    slack_event - A dict of slack event information
 
-    Returns a link to a gif
+    Returns nothing
     """
-    return request.gify_search('gifs', 'kill me')
+
+    # kick user from channel
+    data = {
+        'channel': slack_event["channel"],
+        'user': slack_event['user'],
+    }
+
+    response = json.loads(request.submit_slack_request(data, 'channels.kick', 'USER'))
+    if response['ok']:
+        text = random.choice([
+            'By order of the SDF Armed Forces you have been killed',
+            'Ask not for whom the bell tolls. It tolls for thee ~ John Donne',
+            "We all have been one life to live. Well you do. I'm a bot, and I have backups",
+            'A coward dies a thousand times before thier death, but the valiant \n\
+            taste of death but once.\n ~ William Shakespeare',
+            'Do not go gentle into that good night \n\
+            Old age should burn and rave at close of day; \n\
+            Rage, rage against the dying of the light. \n ~ Dylan Thomas',
+            'Life is cruel. Of this I have no doubt. \n\
+            One can only hope that one leaves behind a lasting legacy.\n\
+            But so often, the legacies we leave behind...are not the ones we intended.\n\
+            ~ Queen Myrrah (Gears of War 2)',
+            "Death is inevitable. Our fear of it makes us play safe, blocks out emotion.\n\
+            It's a losing game. Without passion, you are already dead. \n~Max Payne",
+            "The ending isn’t any more important than any of the moments leading to it.\n\
+            Dr Rosalene (To The Moon)",
+            "Omae wa Mou Shindeiru!",
+            "Stop pitying yourself. Pity yourself, and life becomes an endless nightmare.\n\
+            ~ Osamu Dazai (Bungo Stray Dogs)",
+            "Heghlu’meH QaQ jajvam ~ Klingon Proverb",
+            "batlhbIHeghjaj ~ Klingon Proverb"
+        ])
+    else:
+        text = f"I can't get that done. I get this error: `{response['error']}`"
```

Comment:
A thought: Not for this review / PR, but I imagine this is pretty general behavior across commands? We could re-structure things so we only write the handling code in one spot and it's _always_ treated the exact same way.

Reply:
Sounds like a good idea. Make a Git Issue if you'd like